lib/ui/layers/filters.mjs

```diff
@@ -133,25 +134,20 @@ function filter_null(layer, filter) {
  * @description Generates the minimum and maximum values for numeric filtering.
  * @param {Object} layer - The layer object to apply the filter to.
  * @param {Object} filter - The filter configuration object.
- * @param {string} minmax - The type of value to generate ('min' or 'max').
  * @returns {Promise<void>}
  */
-async function generateMinMax(layer, filter, minmax) {
-  let response = await mapp.utils.xhr(`${layer.mapview.host}/api/query?${mapp.utils.paramString({
-    template: `field_${minmax}`,
+async function generateMinMax(layer, filter) {
+  const response = await mapp.utils.xhr(`${layer.mapview.host}/api/query?${mapp.utils.paramString({
+    template: `field_minmax`,
     locale: layer.mapview.locale.key,
     layer: layer.key,
     table: layer.tableCurrent(),
     field: filter.field,
   })}`);
 
-  // If the response contains response.min or response.max - we can assume the query was successful.
-  // Have to check that response is not null as well
-  response ??= {}
-  if (response[minmax]) {
```

Comment:
Nice one removing this! 

We must still take note that `if(0)` is falsy.

Reply:
This should be resolved.